AODVv2-pandoc-edits/AODVv2.mkd

```diff
@@ -411,12 +411,12 @@ Invalid
     is not referred to as a valid route. Invalid routes contain sequence number information which allows
     incoming information to be assessed for freshness.
 
-If the Local Route Set is stored separately from the Routing Information Base, routes are added to the Routing Information Base
-when LocalRoute.State is valid (set to Active or Idle), and removed from the Routing Information Base when LocalRoute.State becomes Invalid.
+If the Local Route Set is stored separately from the Routing Information Base, routes MAY be added to the Routing Information Base when LocalRoute.State is valid (set to Active or Idle), and MUST be removed from the Routing Information Base when LocalRoute.State becomes Invalid. 
+
+Multiple valid routes for the same address and prefix length but for different metric types may exist in the Local Route Set, but only one route to a particular address and prefix length may exist in the Routing Information Base. The decision of which of the routes should be installed in the Routing Information Base is outside the scope of AODVv2. A LocalRoute should be marked if installed in the Routing Information Base, so that when the LocalRoute changes, the entry in the Routing Information Base can also be updated. 
```

Comment:
It's a guideline for the "out of scope" bit, I guess. What about this? "The decision of the metric type for which routes are installed in the Routing Information Base is outside the scope of AODVv2."